### ISSUE TITLE
Invalidate warnings related to VK_ACCESS_2_NONE

### ISSUE THREAD

**Issue by phuang** [KhronosGroup/Vulkan-ValidationLayers#4682]
In chromium, we saw below warnings. Is it a validate warning? Are VK_ACCESS_2_NONE and VK_ACCESS_NONE same?

[32032:32032:1018/082318.184545:WARNING:angle_platform_impl.cc(48)] RendererVk.cpp:900 (DebugUtilsMessenger): [ UNASSIGNED-BestPractices-ImageBarrierAccessLayout ] Validation Warning: [ UNASSIGNED-BestPractices-ImageBarrierAccessLayout ] Object 0: handle = 0x164400cf4010, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x849fcec7 | vkCmdPipelineBarrier: accessMask is VK_ACCESS_NONE, but for layout VK_IMAGE_LAYOUT_PRESENT_SRC_KHR expected accessMask are VK_ACCESS_2_NONE.
                            Object: 0x164400cf4010 (type = Device(3))

WARN: RendererVk.cpp:900 (DebugUtilsMessenger): [ UNASSIGNED-BestPractices-ImageBarrierAccessLayout ] Validation Warning: [ UNASSIGNED-BestPractices-ImageBarrierAccessLayout ] Object 0: handle = 0x164400cf4010, type = VK_OBJECT_TYPE_DEVICE; | MessageID = 0x849fcec7 | vkCmdPipelineBarrier: accessMask is VK_ACCESS_NONE, but for layout VK_IMAGE_LAYOUT_PRESENT_SRC_KHR expected accessMask are VK_ACCESS_2_NONE.
                            Object: 0x164400cf4010 (type = Device(3))

**Comment by jeremyg-lunarg** [comment]
Yes VK_ACCESS_2_NONE and VK_ACCESS_NONE should be completely equivalent.

What version of the validation layer are you using?

**Comment by jeremyg-lunarg** [comment]
@phuang If you're able to build validation from source, can you please try the attached PR and let me know if it gets rid of these warnings?

**Comment by phuang** [comment]
Thanks for the fix. However the warnings are from our build bots. I didn't reproduce it locally.
